wordpress-mobile/WordPress-Android
# Sharing multiple connections

Fixes #4071 

To test:

Add multiple accounts to a single connection and post. It should post to the multiple accounts correctly.
Also, disconnect accounts to verify updates to accounts are correct.

Needs review: @nbradbury
